zen/core/context.py

```
@@ -170,122 +162,121 @@ def _load_genesis_docs(self):
                     capture_output=True,
-                    text=True
+                    text=True,
                 )
-                self.git_context['remote'] = remote_result.stdout.strip()
-                
-        except Exception as e:
-            console.print(f"[dim]No git context available[/dim]")
-    
+                self.git_context["remote"] = remote_result.stdout.strip()
+
+        except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)